lib/interceptors/not-logged-in.ts

```diff
@@ -0,0 +1,15 @@
+export const onError = async (error) => {
+	const { config, response, request: { responseURL } } = error
+	const { status } = response
+
+	if (status === 401
+		&& response?.data?.message === 'Current user is not logged in'
+		&& config.reloadExpiredSession
+		&& window?.location) {
+		console.error(`Request to ${responseURL} failed because the user session expired. Reloading the page …`)
+
+		window.location.reload()
```

Comment:
Proposal: show a password prompt, instead of reloading the window

Reply:
That is what the reload does, in a way.

There is no API for a login. Currently users have to go through a full page `/login`.

Reply:
> Proposal: show a password prompt, instead of reloading the window

Would be an amazing feature :D